spacewalk/certs-tools/sslToolConfig.py

```diff
@@ -473,8 +473,8 @@ def figureSerial(caCertFilename, serialFilename, indexFilename):
     outstream.close()
     sslerrmsg = ("non-zero exitcode.\n"
                 "If you ran configure-proxy.sh, try copying again the certs from the SUSE Manager Server\n"
-                f"exit-code: {ret}\n"
-                f"error: {sstr(errstream.read())}\n")
+                f"exit-code: {}\n"
+                f"error: {}\n".format(ret, sstr(errstream.read())) )
```

Comment:
This is still containing python3-only features:
* f-strings
* parentheses concatenation

A proper fix would be:
```python
    sslerrmsg = "non-zero exitcode.\n" \
                "If you ran configure-proxy.sh, try copying again the certs from the SUSE Manager Server\n" \
                "exit-code: {}\n" \
                "error: {}\n".format(ret, sstr(errstream.read()))
```